en-US/B.xml

```diff
@@ -16,7 +16,7 @@
 					<emphasis>adj.</emphasis> Hyphenate. For example, "when the back-end database processes a search operation …"
 				</para>
 				 <para>
-					Do not use "backend".
+					Use "backend", as one word with no hyphen, only if it is part of the established product terminology, for example "Mobile Backend-as-a-Service", and when it is not being used to describe a generic process.
```

Comment:
In most(?) other examples we say "Use the one-word form..." It's self-evident that "backend" is one word and doesn't contain a hyphen. You might want to follow that example.